src/roswire/ros2/reader.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+import shlex
+import typing
+from typing import Any, Dict, Optional, Sequence, Tuple
+
+import attr
+import pkg_resources
+from loguru import logger
+
+from ..proxy.roslaunch.config import ExecutableType, LaunchConfig, NodeConfig
+from ..proxy.roslaunch.context import LaunchContext
+from ..proxy.roslaunch.reader import LaunchFileReader
+
+if typing.TYPE_CHECKING:
+    from ..app import AppInstance
+
+
+@attr.s(eq=False)
+class ROS2LaunchFileReader(LaunchFileReader):
+
+    app_instance: 'AppInstance'
```

Comment:
I'm not sure what this is supposed to be?

Reply:
Fixed this.